### PR TITLE
fix last super user removal

### PR DESCRIPTION
**Pull request by joigno**
### Purpose for this PR

Closes https://github.com/windranger-io/windranger-treasury/issues/333

Let me know if someone wants another test or tests. I only included one more test.

### CODE REVIEW

File: contracts/RoleAccessControl.sol [hunk 1]
```diff
@@ -111,6 +114,8 @@ abstract contract RoleAccessControl is RoleMembership {
 
     function revokeSuperUserRole(address account) external onlySuperUserRole {
         _revokeGlobalRole(Roles.SUPER_USER, account);
+        require(_superUserCounter > 1, "revoking last SuperUser");
```
Comment (CjHare) [inline contracts/RoleAccessControl.sol:117]: issue: align the error message with the parent contract error messages https://github.com/windranger-io/windranger-treasury/blob/main/contracts/RoleMembership.sol

As there just a single case, prefixing the contract name, or an abbreviation would be fine.

**Review by CjHare — COMMENTED** [review]
Just the error message, the test looks good 👍

File: contracts/RoleAccessControl.sol (hunk 1 above)
Comment (joigno) [reply contracts/RoleAccessControl.sol:117]: Okey @CjHare, you mean an error msg "RoleAccess: last SuperUser", something like that, that is okey with the linter?

Comment (CjHare) [reply contracts/RoleAccessControl.sol:117]: Yeah something like `RoleAccessControl: no revoking last SuperUser`

Shorter version can include abbreviations for the contract
`RAC: no revoking last SuperUser`

Comment (joigno) [reply contracts/RoleAccessControl.sol:117]: okey abbreviations gotcha.

Comment (joigno) [reply contracts/RoleAccessControl.sol]: error msg improved @CjHare

**Review by CjHare — APPROVED** [review]
Messages look good 👍

**Review by CjHare — APPROVED** [review]

**Comment by aodhgan** [comment]
@joigno pls rebase and merge!

**Pull request merged**